bedrock/mozorg/templates/mozorg/rise25/landing.html

```diff
@@ -0,0 +1,243 @@
+        {% include "mozorg/rise25/includes/social-share.html" %}
+      </div>
+
+      <div class="r25-promote-video">
+        <video class="video" preload="auto" playsinline autoplay loop muted poster="{{ static('img/mozorg/rise25/winners-poster-high-res.jpg') }}" width="450" height="450">
+          <source src="https://assets.mozilla.net/rise25/rise25-2023-winners.webm" type="video/webm">
+        </video>
+
+        {{ resp_img(
+          url='img/mozorg/rise25/winners-poster.jpg',
+          srcset={
+            'img/mozorg/rise25/winners-poster-high-res.jpg': '1.5x'
+          },
+          optional_attributes={
+            'width': '450',
+            'height': '450',
+            'class': 'fallback',
+            'alt': ''
+          }
+        ) }}
+
+        <p class="r25-c-past-cta"><a href="https://rise25.mozilla.org/{{ utm_params }}" rel="external noopener" target="_blank">See last year’s Rise25 winners</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section id="nominate" class="mzp-l-content r25-nominate">
+    <h2 class="r25-section-title">Nomination Categories</h2>
+
+    <div id="advocates" class="r25-c-category r25-t-red">
+      <div class="r25-c-category-body">
+        <h3 class="r25-c-category-title">Advocates</h3>
+        <p class="r25-c-category-desc">Guiding AI towards a responsible future.</p>
+        <p class="r25-c-category-expand">
+          These are the policymakers, activists and thinkers ensuring AI is
+          developed ethically, inclusively and transparently. This category also
+          includes those who are adept at translating complex AI concepts for the
+          broader public — including journalists, content creators and cultural
+          commentators. They champion digital rights and accessible AI, striving
+          to make AI a force for societal good.
+        </p>
+      </div>
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
+      </p>
+    </div>
+
+    <div id="builders" class="r25-c-category r25-t-teal">
+      <div class="r25-c-category-body">
+        <h3 class="r25-c-category-title">Builders</h3>
+        <p class="r25-c-category-desc">Developing AI through ethical innovation.</p>
+        <p class="r25-c-category-expand">
+          They’re the architects of trustworthy AI, including engineers and data
+          scientists dedicated to the principles of open source, open data and
+          open science. They focus on technical proficiency and responsible and
+          ethical construction. Their work ensures AI is secure, accessible and
+          reliable, aiming to create tools that empower and advance society.
+        </p>
+      </div>
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
```

Comment:
**Suggestion (re-iterating Alex's review):** I'm afraid of the lack of color contrast not being obvious to users that this is a CTA, especially if a user's phone screen is dimmer. I noticed you added a slight `box-shadow` property, but maybe something that's a bit more obvious could really help here? i.e. `border: 1px solid white` (definitely consult with designer here, that border property was just a suggestion) 

![image](https://github.com/mozilla/bedrock/assets/42309026/e95b4a07-0947-4c01-b39e-de6b77314b6e)

Reply:
I did add the background gradient which helps but I think it could use even more. I considered making them white, and the gradient would appear on hover, but I didn't want to get too bold without hearing from Ryan. Maybe we'll change it later and let this go for now.